jobs/generation/Utilities.groovy

```diff
@@ -124,6 +124,17 @@ class Utilities {
                                 // For outerloop runs.
                                 'outer-latest-or-auto':'auto-ubuntu1510-20160131outer'
                                 ],
+                            'Ubuntu16.04' :
```

Comment:
Will do that after the Ubuntu16.04 runs are successful and fully supported with packages.